web/src/engine/osk/src/lengthStyle.ts

```diff
@@ -12,6 +12,24 @@ export class ParsedLengthStyle implements LengthStyle {
   public constructor(style: LengthStyle | string) {
     let parsed: LengthStyle = (typeof style == 'string') ? ParsedLengthStyle.parseLengthStyle(style) : style;
 
+    const fallback: LengthStyle = {
+      val: 1,
+      absolute: false
+    };
+
+    if(typeof style == 'number') {
+      // Addresses #13766 for compiled layouts with numbers in the touch-layout `fontsize` field
+
+      // Do not actually emit console warning; we handle those as "errors" in the mobile apps, and
+      // this is more an issue with the compiled keyboard.
+      parsed = fallback;
```

Comment:
Don't we have to set `parsed.val = style` ?

Reply:
With this change we're treating any number as 100%. I'm ok with that, although IMO it would make sense to treat `12` the same as `'12'`, i.e. as `12pt`.

Reply:
The issue there is that this is on `ParsedLengthStyle`, which also handles width, height, etc attributes.  It's not exclusive to handling font styling.  So, a plain "100%" seems the safest option for all codepaths.